src/helpers/helpers.options.js

```diff
@@ -37,14 +37,10 @@ export function calculateTextAlignment(size, options) {
   return x;
 }
 
-function readValueToProps(value, props, defValue) {
+function readValueToProps(value, keys, defValue) {
```

Comment:
Because this function is only used in `toPosition`, I think it can be as well inlined there.

```js
export function toPosition(value) {
  if (isObject(value)) {
    return {
      x: valueOrDefault(value.x, 'center'),
      y: valueOrDefault(value.y, 'center'),
    };
  }
  value = valueOrDefault(value, 'center');
  return {
    x: value,
    y: value
  };
}
```

Reply:
I agree. I'll update it

Reply:
done